wiki/People/Beatmap_Nominators/General_Information/en.md

```diff
@@ -1,57 +1,49 @@
 
-**This icon represents a nomination reset after the map has been Qualified; a disqualification.** As a Nominator, you are not allowed to use this icon under any circumstances. If this was by a placed by a Quality Assurance Team member referencing concerns from the community, the respective community members must first give permission before allowing the two Nominators to renominate. Like the veto, other Nominators can still nominate in these cases.
+## Additional Beatmap Nominator Functions
 
-### ![flame icon](/wiki/shared/icon/flame.gif) Flame
+In addition to being able to nominate beatmaps or reset their nominations, Beatmap Nominators have other functions as well.
 
-**This icon was once used to qualify approval maps, but is now deprecated.** Use the heart icon for this instead.
+### Veto Mediation
 
-### ![star icon](/wiki/shared/icon/star.gif) Star
+When a veto mediation is initiated, a randomly selected jury of Beatmap Nominators will be selected. If you are one of the selected members for a veto mediation, the NAT will send you a message with additional information on how to vote using the [BN Website](http://bn.mappersguild.com). If you are unable to participate, inform a NAT member so that a different Beatmap Nominator can be chosen for the jury in your place.
 
-**This icon does and means absolutely nothing.** Historically this icon has often been used to show interest in a map, but not necessarily that a nomination would follow. This icon has no Beatmap Discussions (Modding v2) equivalent.
+### Quality Assurance Helpers
 
-### ![nuke icon](/wiki/shared/icon/nuke.gif) Nuke
+The Quality Assurance Helpers are volunteers inside of the Beatmap Nominators who actively work on checking qualified beatmap for issues. They do so by reporting maps on the [Report a Qualified beatmap here!](https://osu.ppy.sh/community/forums/topics/447428) thread. Full Beatmap Nominators may join at any time. 
 
-**Nuke icons are placed on mapsets that are considered to be in an irreparable state, unable to ever be ranked.** As a Nominator you are not allowed to use this icon under any circumstances. This icon has no Beatmap Discussions (Modding v2) equivalent.
+The Quality Assurance Helpers' Trello board can be viewed [here](https://trello.com/b/iEsda45d/quality-assurance-helpers).
 
-Note: Any available icon's Beatmap Discussion equivalent often inherits the rules of the icon, so you may still only place the second nomination as a full member of the Beatmap Nominators, for example.
+<!-- TODO: Add more information to help new Beatmap Nominators get used to the role -->
```

Comment:
remove this + prev line & open issue instead to update this stuff

todo comments are like the extreme backlog or "whenever someone notices this"